less/news.less

```diff
@@ -0,0 +1,206 @@
+@import '~eclipsefdn-solstice-assets/less/quicksilver/jakarta/styles.less';
```

Comment:
I don't believe this line is needed.

Reply:
thank you yep, it's not urgent here, just needed colour variables, but because it's not main less file it works :)

Reply:
done